Client/mods/deathmatch/logic/lua/CLuaManager.cpp

```diff
@@ -72,7 +71,9 @@ bool CLuaManager::RemoveVirtualMachine(CLuaMain* pLuaMain)
         m_pRegisteredCommands->CleanUpForVM(pLuaMain);
 
         // Remove it from our list
-        m_virtualMachines.remove(pLuaMain);
+        const auto iter = std::find(m_VirtualMachines.begin(), m_VirtualMachines.end(), pLuaMain);
+        assert(iter != m_VirtualMachines.end()); // Make sure it exists
```

Comment:
Our auto-formatting does unusual things to inline comments, and this comment position is too sane, so I think the code is formatted incorrectly. Try formatting selection.